ops/testing.py

```diff
@@ -11,8 +11,26 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-"""Infrastructure to build unit tests for charms using the ops library."""
+"""Framework for unit testing charms in a simulated Juju environment.
+
+The module includes:
```

Comment:
Ah, "this module includes" works too -- better than "this module provides".

Reply:
cleaned up.